src/frontend/src/common/components/library/data_table/index.tsx

```diff
@@ -67,7 +67,7 @@ export function defaultTreeCellRenderer({
 
   return (
     <TreeRowContent>
-      <Cell data-test-id={`row-${header.key}`}>{displayData}</Cell>
+      <Cell data-test-id={`row-${header.key}`}>{`${displayData}`}</Cell>
```

Comment:
same here

Reply:
Without this, there is a typescript error.  I think this is because displayData isn't a string.
![Screen Shot 2022-08-22 at 9 51 59 AM](https://user-images.githubusercontent.com/109251328/185976139-91c4743d-d8cb-4807-be57-97428f7018a3.png)